hpcgap/src/gvars.c

```diff
@@ -583,17 +582,6 @@ UInt GVarName (
     return INT_INTOBJ(gvar);
 }
 
-Int FindGVarByValue(Obj value, Int start) {
-  Int max = CountGVars;
```

Comment:
I actually implemented a function like this for exporting GAP knowledge for OpenDreamKit. I implemented it in GAP though.

My point is: it might be useful to have this function, even if it doesn't look immediately so.